ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml

```diff
@@ -70,10 +70,10 @@ Popup {
         StatusBaseText {
             id: networkText
             text: {
-                switch (Web3ProviderStore.currentNetwork) {
-                case Constants.networkMainnet: return qsTr("Mainnet");
-                case Constants.networkRopsten: return qsTr("Ropsten");
-                default: return qsTrId("Unknown")
+                switch (Web3ProviderStore.chainName) {
```

Comment:
why does this show only these two networks statically? Mainnet and Ropsten? what about the others?

Reply:
on browser we decided to have only those 2, similar to how it works with ENS/stickers

Reply:
I didnt know that !! :D